test/keda-reconciler-tests.sh

```diff
@@ -39,10 +39,18 @@ export_logs_continuously
 
 header "Running tests"
 
-go_test_e2e -timeout=1h -run=KafkaSource ./test/e2e_new/... || fail_test "E2E (new) suite failed"
-
 if ! ${LOCAL_DEVELOPMENT}; then
 	go_test_e2e -run=KafkaSource -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E sacura tests failed"
 fi
```

Comment:
One more change (sorry I missed this earlier) - could we also move this to after the `BROKER_CLASS` checks and remove the `KafkaSource` flag as well? Thanks!